src/client/components/Tooltip/TooltipIcon.stories.tsx

```diff
@@ -1,25 +1,28 @@
-import { Story } from '@storybook/react'
 import React from 'react'
-import { Tooltip, TooltipProps } from './Tooltip'
+import { Story } from '@storybook/react'
+
+import { TooltipIcon, TooltipIconProps } from './TooltipIcon'
 
 export default {
-  title: 'Components/Tooltip',
-  component: Tooltip,
+  title: 'Components/TooltipIcon',
+  component: TooltipIcon,
   parameters: {
     backgrounds: {
       default: 'gray100',
     },
   },
 }
 
-const Template: Story<TooltipProps> = (args) => <Tooltip {...args} />
+const Template: Story<TooltipIconProps> = (args) => <TooltipIcon {...args} />
 
 export const Large = Template.bind({})
 Large.args = {
   body: 'For more information, see the full terms and conditions above',
+  placement: 'auto',
```

Comment:
If some _Storybook_ wizard knows how to make this accepts a list with possible values of `placement` props, please let me know 😬 

<img width="1073" alt="Screenshot 2022-03-01 at 15 54 50" src="https://user-images.githubusercontent.com/19200662/156191973-4db8b1ac-2b4f-42eb-80a2-eb48191b4897.png">

Reply:
You add an `argTypes` object to the default export - have a look at `Footer.stories.tsx` for an example :)

Reply:
Noice. Thank you :)